types/fluent-ffmpeg/tsconfig.json

```diff
@@ -1,22 +1,22 @@
 {
-    "compilerOptions": {
```

Comment:
tsconfigs are formatted automatically by `scripts/generate-tsconfigs.ts`, don't change it manually.